### PR TITLE
Created a database from diagram

### PR DESCRIPTION
**Pull request by Lichapa**
In this milestone, The following was done:

- Created a database from a diagram.

### CODE REVIEW

File: schema_from_diagram.sql [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+CREATE TABLE medical_histories_treatments (
+medical_history_id INT,
+treatment_id INT,
+CONSTRAINT fk_medical_history FOREIGN KEY(medical_history_id) REFERENCES medical_histories(id),
+CONSTRAINT fk_treatment FOREIGN KEY(treatment_id) REFERENCES treaments(id)
+)
+
+CREATE TABLE invoices (
+id INT GENERATED ALWAYS AS IDENTITY,
+total_amount,
```
Comment (mmsesay) [inline schema_from_diagram.sql]: - [ ] I believe you forgot to assign a datatype to the `total_amount` column. Please add a datatype here.

File: schema_from_diagram.sql [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+type VARCHAR(100),
+name VARCHAR(100),
+PRIMARY KEY (id)
+);
+
+CREATE TABLE medical_histories_treatments (
+medical_history_id INT,
+treatment_id INT,
+CONSTRAINT fk_medical_history FOREIGN KEY(medical_history_id) REFERENCES medical_histories(id),
+CONSTRAINT fk_treatment FOREIGN KEY(treatment_id) REFERENCES treaments(id)
```
Comment (mmsesay) [inline schema_from_diagram.sql:28]: - [ ] Adding indexes for all foreign keys. 
See example below for `medical_histories_treatments`
Please do it for all other foreign keys.
```sql
CREATE INDEX ON medical_histories_treatments (medical_history_id);
CREATE INDEX ON medical_histories_treatments (treatment_id);
```

**Review by mmsesay — CHANGES_REQUESTED** [review]
Hi @Lichapa & @Azonkeu,

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: schema_from_diagram.sql [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+CREATE TABLE invoice_items (
+id INT GENERATED ALWAYS AS IDENTITY,
+unit_price decimal,
+quantity INT,
+total_price decimal,
+invoice_id INT,
+treatment_id INT,
+PRIMARY KEY (id),
+CONSTRAINT fk_invoice FOREIGN KEY (invoice_id) REFERENCES invoices(id)
+CONSTRAINT fk_treatment FOREIGN KEY (treatment_id) REFERENCES treatments(id)
```
Comment (mmsesay) [inline schema_from_diagram.sql:50]: - [ ] Good job adding the indexes for the `invoice_id`. However, the `invoices_items`table has two foreign keys and you only created Index for the `invoice_id`. Please create for the `treatment_id`.

File: schema_from_diagram.sql [hunk 4]
```diff
@@ -0,0 +1,56 @@
...
+)
+
+CREATE TABLE invoices (
+id INT GENERATED ALWAYS AS IDENTITY,
+total_amount DECIMAL(10,2),
+generated_at TIMESTMAP,
+payed_at TIMESTMAP,
+medical_history_id INT,
+PRIMARY KEY (id),
+CONSTRAINT fk_medical_history FOREIGN KEY (medical_history_id) REFERENCES medical_histories(id)
```
Comment (mmsesay) [inline schema_from_diagram.sql:38]: - [ ] There is a foreign key in this `invoices` table which is `medical_history_id`. Please create an index for it also.

**Review by mmsesay — CHANGES_REQUESTED** [review]
Hi @Lichapa & @Azonkeu, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Required Changes ♻️
_Check the comments under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by AshakaE — APPROVED** [review]
Hi @Lichapa , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**